<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Projects\ASP.NET-master\MVCDOTNET\MVCAppTests\obj\Debug\MVCAppTests.dll" total="3" errors="1" failures="0" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2017-12-17" time="08:30:19">
  <environment nunit-version="3.5.0.0" clr-version="4.0.30319.42000" os-version="Microsoft Windows NT 10.0.10586.0" platform="Win32NT" cwd="C:\Projects\ASP.NET-master\MVCDOTNET\MVCAppTests" machine-name="IE11LTB81XTC2" user="H265456" user-domain="GLOBAL" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="C:\Projects\ASP.NET-master\MVCDOTNET\MVCAppTests\obj\Debug\MVCAppTests.dll" executed="True" result="Failure" success="False" time="96.326" asserts="1">
    <properties>
      <property name="_PID" value="235280" />
      <property name="_APPDOMAIN" value="domain-" />
    </properties>
    <failure>
      <message><![CDATA[One or more child tests had errors]]></message>
      <stack-trace />
    </failure>
    <results>
      <test-suite type="TestSuite" name="MVCApp" executed="True" result="Success" success="True" time="0.075" asserts="0">
        <results>
          <test-suite type="TestSuite" name="Controllers" executed="True" result="Success" success="True" time="0.074" asserts="0">
            <results>
              <test-suite type="TestFixture" name="RegisterControllerTests" executed="True" result="Success" success="True" time="0.070" asserts="0">
                <results>
                  <test-case name="MVCApp.Controllers.RegisterControllerTests.UserAddTest" executed="True" result="Success" success="True" time="0.059" asserts="0" />
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
      <test-suite type="TestSuite" name="MVCAppTests" executed="True" result="Failure" success="False" time="96.305" asserts="1">
        <failure>
          <message><![CDATA[One or more child tests had errors]]></message>
          <stack-trace />
        </failure>
        <results>
          <test-suite type="TestSuite" name="UITest" executed="True" result="Failure" success="False" time="96.305" asserts="1">
            <failure>
              <message><![CDATA[One or more child tests had errors]]></message>
              <stack-trace />
            </failure>
            <results>
              <test-suite type="TestSuite" name="TestCases" executed="True" result="Failure" success="False" time="96.305" asserts="1">
                <failure>
                  <message><![CDATA[One or more child tests had errors]]></message>
                  <stack-trace />
                </failure>
                <results>
                  <test-suite type="TestFixture" name="WUITest" executed="True" result="Failure" success="False" time="96.228" asserts="1">
                    <failure>
                      <message><![CDATA[One or more child tests had errors]]></message>
                      <stack-trace />
                    </failure>
                    <results>
                      <test-case name="MVCAppTests.UITest.TestCases.WUITest.Can_Navigate_To_GoToHome" executed="True" result="Error" success="False" time="61.975" asserts="0">
                        <failure>
                          <message><![CDATA[OpenQA.Selenium.WebDriverException : The HTTP request to the remote WebDriver server for URL http://localhost:64942/session timed out after 60 seconds.
  ----> System.Net.WebException : The request was aborted: The operation has timed out.]]></message>
                          <stack-trace><![CDATA[   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)
   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)
   at OpenQA.Selenium.Remote.DriverServiceCommandExecutor.Execute(Command commandToExecute)
   at OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.Remote.RemoteWebDriver.StartSession(ICapabilities desiredCapabilities)
   at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(ICommandExecutor commandExecutor, ICapabilities desiredCapabilities)
   at OpenQA.Selenium.Chrome.ChromeDriver..ctor(ChromeOptions options)
   at MVCAppTests.UITest.Common.Driver.Initialize() in C:\Projects\ASP.NET-master\MVCDOTNET\MVCAppTests\Controllers\UITest\Common\Driver.cs:line 31
--WebException
   at System.Net.HttpWebRequest.GetResponse()
   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)]]></stack-trace>
                        </failure>
                      </test-case>
                      <test-case name="MVCAppTests.UITest.TestCases.WUITest.Can_Navigate_To_Login" executed="True" result="Success" success="True" time="34.253" asserts="1" />
                    </results>
                  </test-suite>
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>